apps/video/js/thumbnail_list.js

```diff
@@ -47,7 +47,8 @@ function ThumbnailList(groupClass, container) {
 
   // If the locale changes while we're running, then we need to
   // re-localize the dates in the group headers
-  navigator.mozL10n.ready(this.localize.bind(this));
+  this.localize();
+  document.addEventListener('DOMRetranslated', this.localize.bind(this));
```

Comment:
Same issue here as in gallery. I'm not convinced that the new code does the same thing as the old code.